docs/matchers/dialogflow.md

```diff
@@ -1,13 +1,13 @@
-# api.ai Matcher
+# Dialogflow Matcher (previously api.ai)
```

Comment:
According to [wikipedia](https://en.wikipedia.org/wiki/Dialogflow) the correct spelling is `Api.ai` with the capital `A` at the beginning. This could be a good opportunity to get this consistent throughout.